tests/e2e/scripts/helm_keadm_e2e.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+KUBEEDGE_ROOT=$PWD
+WORKDIR=$(dirname $0)
+HELM_DIR=$KUBEEDGE_ROOT/build/helm
+E2E_DIR=$(realpath $(dirname $0)/..)
+IMAGE_TAG=$(date +%Y%m%d)
+
+function cleanup() {
+  sudo pkill edgecore || true
+  helm uninstall cloudcore -n kubeedge && kubectl delete ns kubeedge  || true
+  kind delete cluster --name test
+  sudo rm -rf /var/log/kubeedge /etc/kubeedge /etc/systemd/system/edgecore.service $E2E_DIR/keadm/keadm.test $E2E_DIR/config.json
+}
+
+function build_keadm() {
+  make all WHAT=keadm
+  cd $E2E_DIR
+  ginkgo build -r keadm/
+}
+
+function prepare_cluster() {
+  kind create cluster --name test
+
+  echo "wait the control-plane ready..."
+  kubectl wait --for=condition=Ready node/test-control-plane --timeout=60s
+
+  kubectl create clusterrolebinding system:anonymous --clusterrole=cluster-admin --user=system:anonymous
+
+  # edge side don't support kind cni now, delete kind cni plugin for workaround
+  # kubectl delete daemonset kindnet -nkube-system
```

Comment:
this line is commented out? why.

Reply:
In our history tests, it blocks the connection as the comment said "edge side don't support kind cni now". With cloudcore deployed with container type, I think the kind cni would not block it. In additional, I consider using the iptablemgr component outside, in that case, we would use service and the cloudcore will not share hostnetwork.